### ISSUE TITLE
FEATURE REQUEST scrollToTime

### ISSUE THREAD

**Issue by wulfsystems** [stefanuebe/vaadin_fullcalendar#118]
Hey!
Thank you for your work! :)

Will you add 
https://fullcalendar.io/docs/scrollToTime
this in some time?

**Comment by stefanuebe** [comment]
We'll try to integrate it in 4.2. Up to then you can use the java side Element api, to call the method manually, e.g. something like this
```
calendar.getElement().executeJs("this.getCalendar().scrollToTime('10:00')");
```

**Comment by aetasoul** [comment]
Working on a custom class to integrate the Duration object

**Comment by stefanuebe** [comment]
Will be released with the upcoming 4.1.3.

**Comment by stefanuebe** [comment]
Released with 4.1.3.
